BuildPackages.cmd

```diff
@@ -13,7 +13,14 @@ call :BuildNuGetPackage Sarif.Converters %Version% || goto :ExitFailed
 
 ::Build pre-release packages
 call :BuildNuGetPackage Sarif.Driver    %Version%-beta || goto :ExitFailed
-call :BuildNuGetPackage Sarif.Multitool %Version%-beta || goto :ExitFailed
+
+::Build Multitool package from nuspec
+echo .
+echo Building Sarif.Multitool package...
```

Comment:
Can `:BuildNuGetPackage` not do this? Why do we need an explicit command here?

Reply:
Are you saying we should change the package build approach over to nuspecs for all of the projects?

Reply:
Sorry, I didn't understand your change until I looked more carefully at the original file. I now understand that whereas before, we built all four packages by calling the `:BuildNuGetPackage` routine (which invoked `dotnet pack`), now you have modified one of them to use `NuGet.exe` instead, so that you can specify a .nuspec file.

I'm not wild about the resulting structure of the script. We build three packages by calling a subroutine, and we build the fourth with "inline" code. Perhaps instead you could define a second subroutine `:BuildNuGetPackageFromNuspec`, and call that one to build the Multitool package?

I wasn't suggesting that you use .nuspecs for all the packages -- although now that you mention it, it _would_ be nice to build all the packages in the same way.

Reply:
Sounds good. I'll also file an issue to move to nuspecs (the current best practice approach).